components/basecomponents/Exercise/components/Button.tsx

```diff
@@ -1,7 +1,7 @@
 import { forwardRef } from 'react';
 
 interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
-  text: string | Element;
+  text: string | React.ReactNode;
   px?: string;
```

Comment:
toto bych chtěl právě nahradit/předělat ... udělal jsem to jen jako provizorku

Reply:
No právě jsem tu implementaci chtěl nechat na tobě, možností je určitě více. Co napadlo mě tak udělat dvě komponenty, props text odstranit a použít children, vyextrahovat společné className do konstanty, a ty dvě komponenty (TextButton, IconButton) klidně exportovat z jednoho file.